components/x-teaser/src/LiveBlogStatus.jsx

```diff
@@ -15,6 +15,6 @@ const LiveBlogModifiers = {
 export default ({ status }) =>
 	status && status !== 'closed' ? (
 		<div className={`o-teaser__timestamp o-teaser__timestamp--${LiveBlogModifiers[status]}`}>
-			<span className="o-teaser__timestamp-prefix">{LiveBlogLabels[status]}</span>
+			<span className="o-teaser__timestamp-prefix">{` ${LiveBlogLabels[status]} `}</span>
```

Comment:
Have added the trailing space too for consistency with the `<RelativeTime />` component.